### PR TITLE
Pathogen detection from (direct Nanopore) sequencing data using Galaxy - Foodborne Edition, training update 2024

### PR DESCRIPTION
**Pull request by EngyNasr**
I have updated the training based on the latest update for the used workflow

I kindly ask for a review

### CODE REVIEW

File: topics/microbiome/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.md [hunk 1]
```diff
@@ -158,7 +159,7 @@ We will run all these steps using a single workflow, then discuss each step and
 >
 >    {% snippet faqs/galaxy/workflows_import.md %}
 >
-> 2. Run **Workflow 1:  Nanopore Datasets - Pre-Processing** {% icon workflow %} using the following parameters
+> 2. Run **Workflow 1: Nanopore Preprocessing** {% icon workflow %} using the following parameters
 >
 >    {% snippet faqs/galaxy/workflows_run.md %}
```
Comment (shiltemann) [inline topics/microbiome/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.md:164]: I would put the snippet below the parameters

**Comment by shiltemann** [comment]
Thanks @EngyNasr! This looks good from my side

**Review by shiltemann — APPROVED** [review]

**Comment by EngyNasr** [comment]
@shiltemann Thank you soooo much :) , I was pushing some changes to two of the workflows same time, so I made a conflict by mistake, I applied all your changes in the last push but manually to resolve the conflict

**Review by shiltemann — APPROVED** [review]

**Comment by bebatut** [comment]
Thanks @EngyNasr for all the changes. I will merge now!

**Pull request merged**